integration/fuse/bin/alluxio-fuse

```diff
@@ -42,13 +30,6 @@ check_fuse_jar() {
   fi
 }
 
-set_java_opt() {
```

Comment:
moved to `libexec/alluxio-config.sh` the central place to handle all env var setup